.circleci/config.yml

```
@@ -4,7 +4,7 @@ jobs:
   build:
     parallelism: 1
     docker:
-      - image: cimg/elixir:1.14
+      - image: cimg/elixir:1.18.2-erlang-26.2.1-node
```

```suggestion
      - image: cimg/elixir:1.18.2-erlang-26.2.1
```
Is `-node` meant to be there?